static-assets/scripts/dnd-controller.js

```diff
@@ -277,7 +277,7 @@ crafterDefine(
           valid = false;
           publish.call(me, Topics.START_DIALOG, {
             messageKey: 'contentTypeNotFound',
-            link: 'https://docs.craftercms.org/en/3.1/developers/in-context-editing.html',
+            link: 'https://craftercms.com/docs/3.1/developers/in-context-editing.html',
```

Comment:
Outdated?

Reply:
@sumerjabri  the files with links to 3.1 docs are old/unused files, I don't think that those urls need to be updated. I'll create a ticket to clean those old files and translations